### ISSUE TITLE
Upgrading to mongoDB 6.0 from 4.4 will not work

### ISSUE THREAD

**Issue by motoridersd** [bsmithio/OPNsense-Dashboard#38]
I noticed there been some updates to the versions of the containers, so I updated my compose file to reflect this. Mongodb was not happy. I downgraded back to 4.4 to get everything working again. So for anyone already running a 4.4 mongoDB, don't upgrade to 6.0 directly.

**Comment by bsmithio** [comment]
Are you running the stack on a Pi? Seems MongoDB 5.0+ requires a higher Arm architecture and that could be the problem you came across.

**Comment by motoridersd** [comment]
No, running on a Proxmox container on an i5. I did get it to run on 5.0, but it still doesn't like 6. So for now I'm sticking to 5.0.14
![image](https://user-images.githubusercontent.com/5197858/221991840-a5c81e06-1727-4357-ba63-9f58dbcb9051.png)
